clang/Util.d

```diff
@@ -80,8 +80,17 @@ mixin template CX ()
 }
 
 extern (C) int mkstemps(char*, int);
+extern (C) char* mkdtemp(char*);
 extern (C) int close(int);
 
+private void randstr (char[] slice)
+{
+    import std.random;
+
+    for (uint i = 0; i < slice.length; ++i)
```

Comment:
Hmm, when I added the comment the diff showed that `randstr` was added. Now the diff shows that `randstr` is removed. How is that possible?